api/tests/functional-tests/backend/core/test_evaluation.py

```diff
@@ -885,67 +884,3 @@ def test_count_active_evaluations(
         )
         == 0
     )
-
-
-def test__fetch_evaluations_and_mark_for_deletion(
```

Comment:
do we have coverage elsewhere for what this was testing?

Reply:
The function this was testing was rolled into the `delete_evaluations` function in `backend/core/evaluation.py`.

Reply:
This is covered in a few places but i appended the test to the crud tests [here](https://github.com/Striveworks/valor/pull/659/commits/9930a144304bb5a382b92c2960b1a4d2f068dda5).

Reply:
The actual mark for deletion shouldnt affect anything as deletion is only allowed for evaluations in the `Done` or `Failed` state.

Reply:
added a functional test [here](https://github.com/Striveworks/valor/pull/659/commits/12fb2d447671ad04811b850ba21e215b0c2b2737)